blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -135,13 +141,13 @@ const SmallManualPromo = ({ customFields }) => {
 				<Grid as="article" className={containerClassNames}>
 					{["below", "right"].includes(imagePosition) ? (
 						<>
-							<PromoHeading />
-							<PromoImage />
+							<PromoHeading showHeadline={showHeadline} headline={headline} linkURL={linkURL} newTab={newTab} registerSuccessEvent={registerSuccessEvent} />
+							<PromoImage showImage={showImage} searchableField={searchableField} imageParams={imageParams} linkURL={linkURL} newTab={newTab} registerSuccessEvent={registerSuccessEvent} showHeadline={showHeadline} />
```

Comment:
These props are a bit hard to read, could you put them on their own lines like so?

```suggestion
							<PromoHeading
								showHeadline={showHeadline}
								headline={headline}
								linkURL={linkURL}
								newTab={newTab}
								registerSuccessEvent={registerSuccessEvent}
							/>
							<PromoImage
								showImage={showImage}
								searchableField={searchableField}
								imageParams={imageParams}
								linkURL={linkURL}
								newTab={newTab}
								registerSuccessEvent={registerSuccessEvent}
								showHeadline={showHeadline}
							/>
```